templates/explore/repo_search.tmpl

```diff
@@ -1,4 +1,25 @@
-<div class="ui right floated secondary filter menu">
+<div class="ui right floated secondary filter menu" style="z-index: 1">
```

Comment:
```suggestion
<div class="ui right floated secondary filter menu" ">
```
`style="z-index: 1` can you this adjust some how in css files in class

Reply:
Your suggestion has unbalanced `"`. Also I agree we shoud ideally not need this z-index adjustment at all. What is the exact problem?